pyproject.toml

```diff
@@ -144,23 +146,26 @@ crds = "crds"
   "specs/*.json",
 ]
 
-[tool.distutils.nosetests]
-verbosity = 2
-detailed-errors = 1
-where = "crds"
-with-doctest = 1
-doctest-tests = 1
-doctest-options = "+ELLIPSIS,+NORMALIZE_WHITESPACE,+IGNORE_EXCEPTION_DETAIL"
-logging-clear-handlers = 1
+#[tool.distutils.nosetests]
```

Comment:
Unless there is reason to believe these will be needed in the near future, go ahead and remove these lines completely.